extensions/amp-form/0.1/test/test-form-dirtiness.js

```diff
@@ -0,0 +1,183 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {DIRTINESS_INDICATOR_CLASS, FormDirtiness} from '../form-dirtiness';
+
+function getForm(doc) {
+  const form = doc.createElement('form');
+  form.setAttribute('method', 'POST');
+  doc.body.appendChild(form);
+
+  return form;
+}
+
+function changeInput(element, value) {
```

Comment:
nit: let's put helper methods at the end of tests, unless you found that's the pattern in runtime code.